### PR TITLE
WRR-9115: `Scroller`: Fixed to focus properly when the spottable node is bigger than the viewport by scrolling with voice control

### PR DESCRIPTION
**Pull request by mmyelyn**
### Checklist

* [X] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [X] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
When scrolling down with voice control in certain apps, focus is not restored.
Currently, when scrolling with voice control, focus is moved only when the bounds of the viewport include all bounds of the node.

Therefore, it is necessary to handle the case where the node's bounds are not all included in the viewport's bounds.
(when there is a large node that goes out of bounds, the upper part is visible but the lower part is cut off, etc.)

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Added focus logic for cases where the top of the node is long and the bottom of the node is long using node Bounds and viewports Bounds. Since I need to receive and check new nodes, I implemented it using a recursive function.

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRR-9115

### Comments
Enact-DCO-1.0-Signed-off-by: Hyelyn Kim (myelyn.kim@lge.com)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,12 @@
 
 The following is a curated list of changes in the Enact sandstone module, newest changes on the top.
 
+## [unreleased]
+
+### Fixed
+
+- `sandstone/Scroller` to focus even when the node boundery goes out the viewport by voice control
```
Comment (MikyungKim) [inline CHANGELOG.md:9]:
```suggestion
- `sandstone/Scroller` to focus properly when the spottable node is bigger than the size of viewport by voice control
```

File: useScroll/useEvent.js [hunk 2]
```diff
@@ -449,15 +489,7 @@ const useEventVoice = (props, instances) => {
...
-				const nodes = Spotlight.getSpottableDescendants(scrollContainerNode.dataset.spotlightId);
-				for (let i = 0; i < nodes.length; i++) {
-					const nodeBounds = nodes[i].getBoundingClientRect();
-
-					if (nodeBounds[first] >= viewportBounds[first] && nodeBounds[last] <= viewportBounds[last]) {
-						Spotlight.focus(nodes[i]);
-						break;
-					}
-				}
+				focusSpottableNode(scrollContainerNode.dataset.spotlightId, viewportBounds, false);
```
Comment (MikyungKim) [inline useScroll/useEvent.js:492]: Let's use `getDeepSpottableDescendants` instead of this as we discussed.

Comment (mmyelyn) [reply useScroll/useEvent.js]: That's a concise answer. Thank you

**Review by MikyungKim — APPROVED** [review]
LGTM

**Review by juwonjeong — APPROVED** [review]
LGTM

**Pull request merged**